types/src/v2/response.rs

```diff
@@ -45,6 +45,13 @@ pub struct Response<'a, T> {
 	pub id: Id<'a>,
 }
 
+impl<'a, T> Response<'a, T> {
+	/// Create a new `Response`.
```

Comment:
```suggestion
	/// Create a new [`Response`].
```

Same for the others.